### DISCUSSION TITLE
who is the GOAT ? Messi or Ronaldo

### DISCUSSION THREAD

**Discussion by AboodJamal** [community/community#134625] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

1 for Messi
2 for Ronaldo

**Comment by AmroEid77** [comment]
looking for badge

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/134626#discussioncomment-10246867) for reference on why this discussion was closed and locked. Thanks!
